lib/wickedbase.pm

```diff
@@ -254,20 +254,23 @@ sub do_mutex {
     my ($self) = @_;
     my $mutex_name = 'test_' . $self->{name} . '_ready';
     if (check_var('IS_WICKED_REF', '1')) {
+        record_info('mutex wait', $mutex_name);
```

Comment:
`mutex_wait` method does record_info itself - see *"Paused $time"* messages

Reply:
example: https://openqa.suse.de/tests/2236435#step/t01_standalone_card_ifdown_ifreload/38

but for mutex create we have to keep it anyway.

Reply:
thanks for the hint but we actually like how it looks right now :)